sbi/simulators/linear_gaussian.py

```diff
@@ -5,31 +5,24 @@
 from torch.distributions import Independent, MultivariateNormal, Uniform
 
 
-def linear_gaussian(theta: Tensor, std=1.0) -> Tensor:
+def standard_linear_gaussian(theta: Tensor, std=1.0) -> Tensor:
```

Comment:
quite true. How about `diagonal_linear_Gaussian()`?

Reply:
Ok, if you want to talk about the covariance structure, 'isotropic' would be it for a single multiple of the diagonal, else 'diagonal'. Is there a particular reason why we don't use full covariance matrices?

Reply:
full covariance matrizes are allowed in the new `linear_gaussian()` (the one using cholesky). I kept this super simple simulator `diagonal_linear_gaussian()` because it is a bit less verbose in the tests, since we do not have to initialize e.g. the `likelihood_shift` or the `likelihood_cov` to run this simulator. So it is still used for API tests that do not check the `c2st`